app/[userID]/(User)/desktop/Info/index.tsx

```diff
@@ -218,15 +228,16 @@ export default function Info({
             </div>
             <div className={styles.space}></div>
             {
-              (dataType === "sig")
+              (dataType === "sig" && accountData?._id !== "652d60b842cdf6a660c2b778")
```

Comment:
Using `name` may help better identify what this is.
```suggestion
              (dataType === "sig" && accountData?.name !== "公告")
```